packages/fast-tooling-react/src/form/controls/control.select.tsx

```diff
@@ -29,15 +30,20 @@ class SelectControl extends React.Component<
         const {
             selectControl,
             selectControl__disabled,
+            selectControl__default,
             selectControl_input,
         }: SelectControlClassNameContract = this.props.managedClasses;
 
         return (
             <span
-                className={classNames(selectControl, [
-                    selectControl__disabled,
-                    this.props.disabled,
-                ])}
+                className={classNames(
+                    selectControl,
+                    [selectControl__disabled, this.props.disabled],
+                    [
+                        selectControl__default,
+                        isDefault<unknown[]>(this.props.value, this.props.default),
```

Comment:
Do you need to enforce the type here? This fn should infer the types

Reply:
Derp, I don't.